WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterView.xaml

```diff
@@ -23,7 +24,9 @@
             </StackPanel>
           </Button>
         </StackPanel>
-        <controls:MultiTextBox Text="{Binding TransactionString, Mode=TwoWay}" PasteOnClick="True" IsReadOnly="{Binding IsBusy}" Margin="0 8" Watermark="Click here to copy from clipboard!" TextWrapping="Wrap" AcceptsReturn="True" />
+        <Grid ScrollViewer.VerticalScrollBarVisibility="Auto">
```

Comment:
we can remove this ` ScrollViewer.VerticalScrollBarVisibility="Auto"` since we dont have any scrollviewer down the line

Reply:
ACK